<launch>
    <!-- <node pkg="rviz" type="rviz" name="rviz" args="-d $(find map_generator)/rviz/display_grid_map_2d.rviz" /> -->
    
    <!-- 在launch文件里面设置参数服务器的参数 size单位是m-->
    <node pkg="map_generator" type="map_generator_2d" name="map_generator_2d" output="screen" />
        <param name="map/init_x" value="0.0" />
        <param name="map/init_y" value="0.0" />

        <param name="map/x_size" value="10.0" />
        <param name="map/y_size" value="10.0" />
        <param name="map/obstacle_w_lower" value="0.3" />
        <param name="map/obstacle_w_upper" value="1.0" />

        <param name="map/obstacle_number" value="100" />
        <param name="map/resolution" value="0.05f" />
        <param name="map/sensor_rate" value="0.5f" />
    </node>
<launch>








</launch>